bin/evaluate.sh

```diff
@@ -2,6 +2,13 @@
 
 set -o errexit
 
+backtrace="--backtrace"
+if [ "${*#*$backtrace}" != "$*" ]; then
+    export RUST_BACKTRACE=1
+    set -- ${*%$backtrace*}${*#*$backtrace}
```

Comment:
Hmm, I just realized something... the caller(`webpy`) can actually call `playpen` with the shell(`dash`, not bash) and pass it the script, but the best part is that, this way it could also set the env var, like so:
`playpen ppargshere -- /usr/bin/dash -c 'export RUST_BACKTRACE=1; evaluate.sh`
or
`playpen ppargshere -- /usr/bin/dash -c 'RUST_BACKTRACE=1 evaluate.sh`

(this I mentioned like 1 day ago in another issue and I forgot about it completely, see "EDIT3" [here](https://github.com/rust-lang/rust-playpen/issues/119#issuecomment-202261425) for the source of this)

So the question is, should I change this PR to do this?! it seems much better than having to pass(then remove) the `--backtrace` arg!